### PR TITLE
feat: replace deprecated typography classes

### PR DESCRIPTION
**Pull request by yinm**
close: https://github.com/vuetifyjs/eslint-plugin-vuetify/issues/20

I added rules for v2.3.0 typography classes.

### CODE REVIEW

File: lib/rules/no-deprecated-classes.js [hunk 1]
```diff
@@ -23,7 +23,19 @@ const replacements = new Map([
   ['no-wrap', 'text-no-wrap'],
   ['ellipsis', 'text-truncate'],
   ['left', 'float-left'],
-  ['right', 'float-right']
+  ['right', 'float-right'],
+  ['display-4', 'text-h1'],
```
Comment (jacekkarczmarczyk) [inline lib/rules/no-deprecated-classes.js:27]: This should not replace 2.3 classes if someone is on the vuetify 2.0-2.2

Comment (yinm) [reply lib/rules/no-deprecated-classes.js:27]: Oh, I see... I'll think solution for this.

Comment (yinm) [reply lib/rules/no-deprecated-classes.js]: I rebased to master branch and added new rule for replacing typography classes. 
How do you think it?

Comment (jacekkarczmarczyk) [reply lib/rules/no-deprecated-classes.js]: tbh i think that the best solution would be detecting what version of vuetify is installed

File: lib/rules/no-deprecated-classes.js [hunk 2]
```diff
@@ -29,6 +30,21 @@ const replacements = new Map([
   // [/([mp][axytblr])-(\d)/, (type, n) => `${type}-${spacers[n]}`]
 ])
 
+if (getInstalledVuetifyVersion() >= '2.3.0') {
+  replacements.set('display-4', 'text-h1')
```
Comment (KaelWD) [inline lib/rules/no-deprecated-classes.js:34]: `.set()` can be chained like `map.set().set()`

File: tests/lib/rules/no-deprecated-classes.js [hunk 3]
```diff
@@ -15,6 +15,18 @@ tester.run('no-deprecated-classes', rule, {
     '<template><div class="multiple classes" /></template>',
     '<template><div class="justify-center" /></template>',
     '<template><v-layout text-center /></template>',
+    '<template><div class="text-h1" /></template>',
```
Comment (KaelWD) [inline tests/lib/rules/no-deprecated-classes.js:18]: Don't need a test for every possible value, all the other tests in here are just making sure the fixer works in every scenario (class, attribute, v-bind, regexp replacer etc.)

**Comment by yinm** [comment]
Thanks your review 🙏 
I reflected it.

**Review by KaelWD — APPROVED** [review]

**Pull request merged**